analytics/consumer.py

```diff
@@ -22,15 +20,15 @@ class Consumer(Thread):
     """Consumes the messages from the client's queue."""
     log = logging.getLogger('segment')
 
-    def __init__(self, queue, write_key, flush_at=100, host=None,
```

Comment:
BATCH_SIZE_LIMIT above does not seem correct we support 4mb right?

Reply:
change logger from segment to rudder, all segment refferences should be replaced

Reply:
is MAX_MSG_SIZE value correct for our services?